pootle/core/views.py

```diff
@@ -464,7 +465,8 @@ def is_admin(self):
 
     @property
     def language(self):
-        return None
+        if self.tp:
+            return self.tp.language
```

Comment:
its not at all "implicit" this is the parent class - and this is a method of the all of the descendants